insights/specs/default.py

```diff
@@ -115,6 +115,7 @@ class DefaultSpecs(Specs):
     compliance_policies = compliance_ds.compliance_policies
     compliance_assign = compliance_ds.compliance_assign
     compliance_unassign = compliance_ds.compliance_unassign
+    compliance_advisor_rule_enabled_policies = compliance_ds.compliance_advisor_rule_enabled
```

Comment:
As this is not a spec belong to the `--compliance` family, please move it to the next section.